### ISSUE TITLE
Comments in tslint.json are flagged as errors

### ISSUE THREAD

**Issue by egamma** [Microsoft/vscode-typescript-tslint-plugin#8]
Testing: https://github.com/Microsoft/vscode/issues/62092

The tslint.json may include comments, but they are flagged as errors.

![image](https://user-images.githubusercontent.com/172399/47846482-4673ec00-ddc8-11e8-9dc7-5296fa45bdf9.png)

The extension should associate tslint.json with the `jsonc` language (JSON with comments).

```
  "contributes": {
    "languages": [
      {
        "id": "jsonc",
        "filenames": [
          "tslint.json"
        ]
      }
    ],
```
